addons/hr_expense/models/hr_expense.py

```diff
@@ -275,6 +281,15 @@ def action_get_attachment_view(self):
         res['context'] = {'default_res_model': 'hr.expense', 'default_res_id': self.id}
         return res
 
+    @api.multi
+    def refuse_expense(self, reason):
```

Comment:
Actually we move to action_XXX naming convention when having a workflow-like behavior. So I would rename it action_refuse .By the way, why is it ensure_one, and why don't you store the reason in refuse_reason ?